update_database_scripts/createStudyTable.R

```diff
@@ -1,262 +1,258 @@
-# This script uses data from an existing folder populated with CSV trait data and the GWAS catalog acccessable via the gwasrapidd library 
```

Comment:
Is this mostly the same except for the 'cat(..)' line at the bottom?

Reply:
This was one of the two most changed files. I fixed it so you can see the changes now.